dev-requirements.txt

```diff
@@ -16,7 +16,7 @@ codacy-coverage
 Sphinx==1.6.5
 sphinx_rtd_theme
 flasgger==0.9.1
-PyYAML==3.13
+PyYAML==4.2b1
```

Comment:
if it's in requirements.txt we probably don't need it in dev-requirements.txt

Reply:
🤷‍♂️